src/components/util/util.js

```diff
@@ -137,3 +137,7 @@ export function hasEveryLoanItemStatus(loans, itemStatus) {
 export function hasAnyLoanItemStatus(loans, itemStatuses) {
   return _.every(Object.values(loans), loan => itemStatuses.includes(loan?.item?.status?.name));
 }
+
+export function checkFeesFinesStatus(accounts, status) {
+  return accounts.every((account) => account.status.name.toLowerCase() === status.toLowerCase());
+}
```

Comment:
Maybe rename to something like `accountsMatchStatus()` or `validateAccountsHaveStatus()? Functions named `check...` always kinda bug me because it isn't clear what happens if the check succeeds (nothing? side effect? return true?) or fails (throw? return false?). NBD though; just an observation.

Reply:
`accountsMatchStatus` sounds good. Renamed method